src/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List - WebPack</title>
+    <link rel="icon" type="image/png" href="./logo.png" />
+    <script
+      src="https://kit.fontawesome.com/3848942bcd.js"
+      crossorigin="anonymous"
+    ></script>    
```

Comment:
Please, it would be great if you name your web page just as To Do List, many users don't have to know that this exercise was doing using webpack.

Reply:
Hi @oovillagran, I make the changes suggested.

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/index.html#L6

Reply:
Hi @oovillagran, ✋  I create a class to refactor the use of `flex`:

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/style.css#L34

And use in the `HTML`:

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/index.html#L20

And JavaScript code:

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/logic.js#L27

Thanks for your observations. 😄